File: 310971/Guzik/task-320/src/main/java/com/example/entitiesapp/entities/Sticker.java
package com.example.entitiesapp.entities;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;
import java.util.HashSet;
import java.util.Set;

@Entity
@Table(name = "tbl_sticker")
@Getter
@Setter
public class Sticker extends BaseEntity {
    @Column(name = "name", unique = true, nullable = false, length = 32)
    private String name;

    @ManyToMany(mappedBy = "stickers")
    private Set<Article> articles = new HashSet<>();
}

File: 351004/Kuchko/ServerApp/ServerApp/Models/Entities/Sticker.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace ServerApp.Models.Entities;

[Table("tbl_sticker")]
public class Sticker : BaseEntity
{
    [Column("name")] public string Name { get; set; } = null!;

    public virtual ICollection<Article> Articles { get; set; } = new List<Article>();
}